modules/core/packet-server/client/cli/query.go

```diff
@@ -13,12 +13,12 @@ import (
 	"github.com/cosmos/ibc-go/v9/modules/core/packet-server/types"
 )
 
-// getCmdQueryClient defines the command to query the client information (creator and counterparty) for the given client ID.
-func getCmdQueryClient() *cobra.Command {
+// getCmdQueryChannel defines the command to query the client information (creator and channel) for the given client ID.
+func getCmdQueryChannel() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "client [client-id]",
```

Comment:
will need to update these also within v2. I don't think there's any point in refactoring them in packet-server as it will eventually be removed, right?

happy to create an issue for CLI's and queries in v2 pkg.